Resources/doc/configuration.rst

```diff
@@ -95,8 +95,8 @@ Configuration Reference
                         platform_service:     ~
                         auto_commit:          ~
 
-                        # If set to "/^sf2_/" all tables, and any named objects such as sequences, not prefixed with "sf2_" will be ignored by the schema
-                        # tool. This is for custom tables which should not be altered automatically.
+                        # If set to "/^sf2_/" all tables, and any named objects such as sequences, not prefixed with "sf2_" will be ignored by the schema tool.
```

Comment:
What I mean by wrapping is avoiding long lines by splitting them when reaching a certain length